iD/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:16.04
+ENV workdir /app
+RUN apt-get update -y
+RUN apt-get install -y curl
+RUN curl -sL https://deb.nodesource.com/setup_8.x | bash -
+RUN apt-get install -y nodejs
+RUN apt-get install -y wget python-pip software-properties-common python-software-properties git 
+RUN pip install awscli
+RUN git clone --depth=1 https://github.com/openstreetmap/iD.git $workdir
```

Comment:
Can we pin this to a particular git sha so that we don't get unexpected changes?